crates/oxc_linter/src/snapshots/no_async_await.snap

```diff
@@ -37,3 +37,30 @@ source: crates/oxc_linter/src/tester.rs
  4 │             };
    ╰────
   help: Async/await is not allowed
+
+  ⚠ oxc(no-async-await): Unexpected async/await
+   ╭─[no_async_await.tsx:3:22]
+ 2 │         class Foo {
+ 3 │             async foo() {}
+   ·                      ─────
```

Comment:
😵

Reply:
Yup, not great. Gonna create an issue for this.